debs/siegfried/Dockerfile

```diff
@@ -1,19 +1,25 @@
 FROM ubuntu:xenial
 
+# Environmnet variables needed during build
+ENV GOVERSION 1.11
+ENV GOOS linux
+ENV GOARCH amd64
 
-RUN apt-get update && \
-    apt-get install -y dpkg-dev git build-essential wget debhelper \
-    devscripts equivs dh-golang software-properties-common python-software-properties 
+# Environment variables needed during runtime
+ENV GOPATH /go
+ENV PATH /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/go/bin:/go/bin
 
-RUN add-apt-repository ppa:gophers/archive
-RUN apt-get update && apt-get install -y golang-1.9
+RUN apt-get update && \
+    apt-get install -y dpkg-dev git build-essential wget debhelper curl \
+    devscripts equivs dh-golang software-properties-common python-software-properties && \
```

Comment:
Perhaps you don't need `software-properties-common python-software-properties` since we're not using `add-apt-repository` anymore?

Reply:
You are right about that. I removed it! Thank you!